messages.rb

```diff
@@ -276,6 +289,92 @@ def send_list(event)
   event.attach_file(File::open(tmpfile))
 end
 
+def send_community(event)
+  msg = event.content
+  tabs = parse_tabs(msg)
+  tabs.empty? ? ftabs = "" : ftabs = format_tabs(tabs)
```

Comment:
I think `format_tabs` will take care of leaving this string empty if the array is empty. So no need to check here.

Also, if you're going to assign to a variable in a ternary like this, it's more pleasant to put it after the `=` like `ftabs = tabs.empty? "" : format_tabs(tabs)`. You avoid repeating the variable name, and make it clear that it's an assignment - I didn't realize this declared a new variable for a minute.

Reply:
Oh, you're right. It was giving me problems because, if tabs is empty, I fill it afterwards in order to always be able to use the line "where(tab: tabs)".

I think I'm going to change it to what seems to be more customary in inne, which is to define a static method in the HighScore module that starts with the check  "tabs.empty? ? type.all : type.where(tab: tabs)" and use it to return the tls and the tes. That way the code will be cleaner too, since this method looks a bit convoluted.